assets/app/scripts/controllers/projects.js

```diff
@@ -73,6 +73,17 @@ angular.module('openshiftConsole')
       DataService.list("projects", $scope, function(projects) {
         $scope.projects = projects.by("metadata.name");
         $scope.showGetStarted = hashSizeFilter($scope.projects) === 0;
+
+        // if projectToDelete is in URL and if project exists to be deleted
+        if ($location.search()['projectToDelete']) {
```

Comment:
instead of passing this as a URL parameter, I'd prefer this was just done in memory, could have a flash messages service or something